Assets/Scripts/Spawners/Spawner.cs

```diff
@@ -13,11 +13,16 @@ public abstract class Spawner: MonoBehaviour
     protected abstract void Update();
     protected abstract void spawn();
 
-    public void EnemyDied()
+    public void enemyDied()
```

Comment:
I know it is weird coming from Java, but C# methods naming convention is to follow Pascal case and capitalize the first letter. https://msdn.microsoft.com/en-us/library/4df752aw(v=vs.71).aspx